packages/oui-navbar/src/menu/navbar-menu.controller.js

```diff
@@ -0,0 +1,70 @@
+export default class {
+    constructor ($element, NavbarService, NavbarGroupService, KEYBOARD_KEYS) {
+        "ngInject";
+
+        this.$element = $element;
+        this.navbarService = NavbarService;
+        this.navbarGroupService = NavbarGroupService;
+        this.KEYBOARD_KEYS = KEYBOARD_KEYS;
+    }
+
+    toggleMenu (state, isInternalNav) {
+        // Update navbar navigation
+        this.navigation = this.navbarService.toggleMenu(state, isInternalNav);
+    }
+
+    closeMenu (state, isInternalNav) {
+        // Update navbar navigation
+        this.navigation = this.navbarService.toggleMenu(state, isInternalNav);
+
+        // Set focus on menu toggler, preceding $element
+        const prev = this.$element[0].previousElementSibling;
+        if (prev) {
+            prev.focus();
+        }
+    }
+
+    // Close method for button with click function
+    closeMenuWithCallback ($event, callback) {
+        // Call the callback function
+        if (typeof callback === "function") {
+            callback($event);
+        }
+
+        // Then close all menus
+        this.navigation = this.navbarService.toggleMenu();
+    }
+
+    /* eslint-disable class-methods-use-this */
```

Comment:
Don't really like hiding eslint warn but... the PR about this rule has not been merged yet. (https://github.com/ovh-ux/eslint-config-ovh/pull/9) so... ok!

Reply:
I change to a static method. The comment has been removed